census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -366,12 +365,13 @@ public void streamClosed(io.grpc.Status status) {
       span.end(createEndSpanOptions(status, isSampledToLocalTracing));
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Need to add a TODO saying we will replace `ContextUtils` usage as recommended to get rid of this supressWarnings annotation? Possibly with a reference that describes what to do

Reply:
Added TODO. PTAL